### ISSUE TITLE
Website: dark theme insconsistently applied, making text illegible

### ISSUE THREAD

**Issue by evenreven** [containers/podman-desktop#2380]
### Bug description

This is the frontpage of podman-desktop.io for me:

![bilde](https://user-images.githubusercontent.com/3036077/236782464-400d43ca-4e95-43d3-b716-233a0a02bab0.png)

It's not just this, the OS illustration SVGs are light-on-light etc. (Obviously this violates WCAG guidelines for contrast.)

It seems from scanning the HTML output, that in the CSS the light mode style for the background (`bg-zinc-100` in this case) wins (although `data-theme` is set to `dark` on the `html` element), but the dark mode style (`dark:text-white`) wins for the text, making both nearly white. I'm not a Tailwind expert, so I don't know where the bug is, but changing the `bg-zinc-100` class name to `light:bg-zinc-100` fixes the bug, so it could be as simple as being explicit about the mode in the classes for the section node.

Tested on Firefox for Android, Edge for Android, Firefox for Windows, and Edge for Windows - all exhibit the same behaviour. (All are set to light mode, but it looks like podman-desktop.io doesn't use system settings anyway, so it probably doesn't matter.)

Setting the website to light mode manually makes the text dark as intended. Setting it back to dark breaks the design again.

### Operating system

Not applicable

### Version

0.15.0

**Comment by benoitf** [comment]
thanks for the report @evenreven 
it looks like an update has broken the website

**Comment by evenreven** [comment]
A suggestion if looking for the bug takes time: set light mode as default in the meantime.

**Comment by benoitf** [comment]
will be fixed by
https://github.com/containers/podman-desktop/pull/2394

and for system settings:
https://github.com/containers/podman-desktop/pull/2395

**Comment by benoitf** [comment]
@evenreven you may check the fixed version of the Pull Request here: https://6459f67b97c52634b1d8b6ce--podman-desktop-pr.netlify.app/
